Tests/Command/expected/config.js

```diff
@@ -35,15 +35,15 @@
     });
 
     app.config(function($provide, NgAdminConfigurationProvider) {
-        $provide.factory("PostAdmin", function() {
+        $provide.factory("PostsAdmin", function() {
             var nga = NgAdminConfigurationProvider;
-            var post = nga.entity('post');
+            var posts = nga.entity('posts');
```

Comment:
Would be better to use singular terms for variable name. We define a single post entity. Using a plural here make think we got an array of posts.